src/zui/ZUIOrganizeSidebar/SidebarListItem.tsx

```diff
@@ -77,4 +76,6 @@ const SidebarListItem = forwardRef<HTMLDivElement, SidebarListItemProps>(
   }
 );
 
+SidebarListItem.displayName = 'SidebarListItem';
```

Comment:
Nice! I didn't know you could do it this way. Good solution!

Reply:
I just found out!